packages/fuel-gauge/src/void.test.ts

```diff
@@ -0,0 +1,83 @@
+import { launchTestNode } from 'fuels/test-utils';
+
+import { VoidAbi__factory } from '../test/typegen';
+import type { NativeEnumInput } from '../test/typegen/contracts/VoidAbi';
+import VoidAbiHex from '../test/typegen/contracts/VoidAbi.hex';
+import type { Option } from '../test/typegen/contracts/common';
+
+/**
+ * @group node
```

Comment:
Should be browser compatible
```suggestion
 * @group node
 * @group browser
```

Reply:
2f5b4bb7399a268b9d736ed73bddda15403d0fc2